src/main/java/mat/server/util/MeasureTransferUtil.java

```diff
@@ -0,0 +1,48 @@
+package mat.server.util;
+
+import com.amazonaws.auth.InstanceProfileCredentialsProvider;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.PutObjectResult;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import mat.dto.MeasureTransferDTO;
+import org.apache.commons.lang3.StringUtils;
+
+public class MeasureTransferUtil {
+
+    public static final AmazonS3 buildAwsS3Client() {
+        return AmazonS3ClientBuilder
+                .standard()
+                .withRegion(getMeasureTransferS3Region())
+                .withCredentials(new InstanceProfileCredentialsProvider(true))
+                .build();
+    }
+
+    public static final PutObjectResult uploadMeasureDataToS3Bucket(MeasureTransferDTO ob, String measureId)
+            throws JsonProcessingException {
+        String objectKeyName = "measure_" + measureId;
+        String bucketName = getMeasureTransferS3BucketName();
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        String transferJson = objectMapper.writeValueAsString(ob);
+        return buildAwsS3Client()
+                .putObject(bucketName, objectKeyName, transferJson);
+    }
+
+    public static final String getMeasureTransferS3Region() {
```

Comment:
It's probably clearer, but it probably can be replaced by getSystemProperty() right?

Reply:
yeah maybe.... let me do it. I think doesn't add any value adding a separate method.

Reply:
done